synthesizer/process/src/stack/authorization/mod.rs

```diff
@@ -65,10 +68,21 @@ impl<N: Network> TryFrom<(Vec<Request<N>>, Vec<Transition<N>>)> for Authorizatio
             requests.len(),
             transitions.len()
         );
```

Comment:
A comment at the top of this method documenting why this technique is needed would be helpful. Something along the lines of:
```
Requests are added to an authorization in the pre-order traversal sequence (parents before children), while transitions are added to an authorization in the post-order traversal sequence (children before parents). 
This method takes in as input requests and transitions, in the same order.
```

Reply:
Assuming my understanding is correct :)